python/helpers/pydev/pydevd.py

```diff
@@ -1358,6 +1358,33 @@ class SetupHolder:
     setup = None
 
 
+def apply_debugger_options(setup_options):
+    """
+
+    :type setup_options: dict[str, bool]
+    """
+
+    default_options = {'save-signatures': False, 'save-threading': False, 'save-asyncio': False, 'qt-support': False}
```

Comment:
`save-signatures` and `save-threading` are not debugger options, they are used in concurrency visualisation, and can't be used from console.